libs/MeshKernel/src/Mesh.cpp

```diff
@@ -224,12 +224,12 @@ void Mesh::MergeTwoNodes(UInt firstNodeIndex, UInt secondNodeIndex)
 
     if (firstNodeIndex >= GetNumNodes())
     {
-        throw ConstraintError("The first node-index is out of range: value = {}, number of nodes = {}", firstNodeIndex, GetNumNodes());
+        throw RangeError("The first node-index is out of range: value = {}, number of nodes = {}", firstNodeIndex, GetNumNodes());
```

Comment:
I meant to say use `range_check::CheckGreaterEqual` , which throws a range error.

https://github.com/Deltares/MeshKernel/blob/8bd641bfd5b0bc5a981a3032673bfd0a28dd9fe6/libs/MeshKernel/include/MeshKernel/RangeCheck.hpp#L235

For usage see

https://github.com/Deltares/MeshKernel/blob/master/libs/MeshKernel/tests/src/RangeCheckTests.cpp